netmap/migration_test.go

```diff
@@ -184,3 +215,26 @@ func testMigrationFromDump(t *testing.T, d *dump.Reader) {
 		require.ElementsMatch(t, vPrev, vNew, "%d-th past netmap snapshot should remain", k)
 	}
 }
+
+func checkNewEpochSubscribers(t *testing.T, contract *migration.Contract, balanceWant, containerWant util.Uint160) {
+	require.Nil(t, contract.GetStorageItem(balanceHashOldKey))
+	require.Nil(t, contract.GetStorageItem(containerHashOldKey))
+
+	// contracts are migrated in alphabetical order at least for now
+
+	contract.SeekStorage(append(newEpochSubsNewPrefix, 0), func(k, v []byte) bool {
+		balanceGot, err := util.Uint160DecodeBytesLE(k)
```

Comment:
u dont check func entrance. I'd gather map index->address and check after `SeekStorage` call

Reply:
fixed by bool flags